Automattic/woocommerce-subscriptions-core
# show only valid payment methods when creating/editing a subscription on wp-admin

This is a partial fix to https://github.com/Automattic/woocommerce-payments/issues/2962 and 
fix #29 


#### Changes proposed in this Pull Request
This PR adds an ajax request to load which payment gateways are available for a selected customer when creating a new subscription/editing a subscription.

#### Testing instructions
Case 1
- With WC Payments plugin **active** and WC Subscription **inactive**
- Go to **WooCommerce > Subscriptions > Add new**
- Choose a customer **with** a saved credit card on WCPayments 
- Click the pencil icon in the billing fields section. 
- It should show **only WC Payments**

Case 2
- With WC Payments plugin **active** and WC Subscription **inactive**
- Go to **WooCommerce > Subscriptions > Add new**
- Choose a customer **without** a saved credit card on WCPayments 
- Click the pencil icon in the billing fields section. 
- It should show **only Manual Renewal**

Case 3
- With WC Payments plugin **active** and WC Subscription **active**
- Go to **WooCommerce > Subscriptions > Add new**
- Choose a customer **with** a saved credit card on WCPayments 
- Click the pencil icon in the billing fields section. 
- It should show **Manual Renewal and WC Payments**
